src/YoutubeAtom.test.tsx

```diff
@@ -53,77 +54,11 @@ describe('YoutubeAtom', () => {
                 },
             });
 
-            jest.advanceTimersByTime(3000);
+            jest.advanceTimersByTime(1000);
             expect(eventEmitters[0]).toHaveBeenCalledTimes(1);
             expect(eventEmitters[0]).toHaveBeenCalledWith('play');
         });
 
-        it('should dispatch 25% watched event', () => {
```

Comment:
these tests are flaky, likely due to the fact they depend on setInterval code that is cleared on every itteration